go.mod

```diff
@@ -34,6 +35,7 @@ replace (
 	github.com/containernetworking/cni => github.com/containernetworking/cni v0.8.1
 	github.com/containers/buildah => github.com/containers/buildah v1.23.1
 	github.com/opencontainers/image-spec => github.com/opencontainers/image-spec v1.0.2-0.20210819154149-5ad6f50d6283
+	github.com/project-flotta/flotta-operator => github.com/machacekondra/k4e-operator v0.0.2-0.20220328125045-9563eac90f21
```

Comment:
Should we wait for [PR 137](https://github.com/project-flotta/flotta-operator/pull/137) before merging this?

Reply:
The PR has just been merged.

Reply:
@machacekondra - please update go.mod so we can merge this PR.